real-main/app/models/user/manager.py

```diff
@@ -590,6 +594,16 @@ def on_appstore_transaction_add(self, transaction_id, new_item):
             price = new_item.get('price', Decimal('0'))
             self.dynamo.increment_paid_real_so_far(new_item['userId'], price)
 
+    def on_user_new_followers_sync_card(self, user_id, new_item, old_item=None):
+        followed_user = self.get_user(user_id)
+        followed_user_privacy_status = followed_user.item.get('privacyStatus', UserPrivacyStatus.PUBLIC)
+        follow_status = (new_item or {}).get('followStatus', FollowStatus.NOT_FOLLOWING)
```

Comment:
I think new_item is always defined here?

```suggestion
        follow_status = new_item.get('followStatus', FollowStatus.NOT_FOLLOWING)
```